sale_purchase_force_vendor/models/sale_order_line.py

```diff
@@ -35,6 +35,17 @@ def _compute_vendor_id_domain(self):
             )
             item.vendor_id_domain = domain
 
+    def _prepare_product_supplierinfo_vals(self):
```

Comment:
Too generic name, as other module may create also a product.supplierinfo from here:

```suggestion
    def _prepare_force_vendor_product_supplierinfo_vals(self):
```